components/touchscreen/index.rst

```diff
@@ -19,28 +19,82 @@ Base Touchscreen Configuration
     # Example touchscreen
     touchscreen:
       - platform: ...
+        display: display1
         on_touch:
           then:
             ...
+        on_update:
+          then:
+            ...
+        on_release:
+          then:
+            ...
 
 Configuration variables:
 ************************
+- **display** (*Required*, :ref:`config-id`): The display to use this touchscreen with.
```

Comment:
```suggestion
- **display** (*Required*, :ref:`config-id`): The display to use this touchscreen with. This will be provided automatically if only one display is configured on the device.
```